lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -4,5 +4,9 @@ class MiqAeServicePhysicalServer < MiqAeServiceModelBase
     expose :turn_off_loc_led
     expose :power_on
     expose :power_off
+
+    def self.create_server_profile_and_deploy_task(template_id, server_id, profile_name)
+      PhysicalServerProfileTemplate.deploy_server_from_template(template_id, server_id, profile_name)
```

Comment:
I don't think this is right because this will run on the client side as opposed to the server side.  @agrare Can you comment here?  Maybe I'm wrong but I thought we did these different.

Reply:
@Fryguy what is the right way to do this? we need to call cisco intersight API client functions in the manageiq automation engine and deploy a physical server with the given server profile template

Reply:
fixed